src/more-infos/more-info-group.js

```diff
@@ -36,4 +39,40 @@ export default new Polymer({
       ],
     },
   },
+
+  observers: [
+    'statesChanged(stateObj, states)',
+  ],
+
+  statesChanged(stateObj, states) {
+    let groupDomainStateObj = false;
+
+    if (states && states.length > 0) {
+      const baseStateObj = maxBy(states, (s) => Object.keys(s.attributes).length);
+
+      groupDomainStateObj = baseStateObj.set('entityId', stateObj.entityId).set(
+          'attributes', Object.assign({}, baseStateObj.attributes));
+    }
+
+    for (const s of states) {
```

Comment:
No, , the of iterator depends on symbol(iter)  to exist. Learned that first
hand at $job with ie10

On Tue, May 17, 2016, 17:06 amorsillo <notifications@github.com> wrote:

> In src/more-infos/more-info-group.js
> <https://github.com/home-assistant/home-assistant-polymer/pull/61#discussion_r63626061>
> :
>
> > +
> > +  observers: [
> > +    'statesChanged(stateObj, states)',
> > +  ],
> > +
> > +  statesChanged(stateObj, states) {
> > +    let groupDomainStateObj = false;
> > +
> > +    if (states && states.length > 0) {
> > +      const baseStateObj = maxBy(states, (s) => Object.keys(s.attributes).length);
> > +
> > +      groupDomainStateObj = baseStateObj.set('entityId', stateObj.entityId).set(
> > +          'attributes', Object.assign({}, baseStateObj.attributes));
> > +    }
> > +
> > +    for (const s of states) {
>
> I'm not sure which line you are referring to here. Won't babel take care
> of any browser normalization when it is built anyway?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/home-assistant/home-assistant-polymer/pull/61/files/96231485e313dfd6edeb4fef88287e4422d1b8a5#r63626061>
>